regression/verilog/modules/parameters6.desc

```diff
@@ -0,0 +1,6 @@
+KNOWNBUG
+parameters6.v
+
+^EXIT=0$
+^SIGNAL=0$
+--
```

Comment:
I'd appreciate if "KNOWNBUG" tests include (after a further `--` line) a comment that says what gap in our implementation this test refers to.